pkg/operator/operator.go

```diff
@@ -189,6 +192,46 @@ func New(logger logr.Logger, clientConfig *rest.Config, opts Options) (*Operator
 	if err != nil {
 		return nil, fmt.Errorf("invalid port: %w", err)
 	}
+
+	var ca *x509.Certificate
+	if fileExists(opts.ClientCAFile) {
+		rawCA, err := os.ReadFile(opts.ClientCAFile)
+		if err != nil {
+			return nil, fmt.Errorf("read CA file: %w", err)
+		}
+		ca, err = x509.ParseCertificate(rawCA)
+		if err != nil {
+			return nil, fmt.Errorf("parse CA file: %w", err)
+		}
+	} else if opts.CACert != "" {
```

Comment:
Does it make sense for the user to provide both flags if they're mutually exclusive? Do we not want to show an error or a warning if they put both since 1 is effectively useless? Or will deprecate the inline certs in the near future?

Reply:
Good idea. Yeah, I will log a warning.

But yes, the flags for inline certs should disappear soon after the necessary changes are submitted/released on the GKE side.